src/tests/ftest/security/container_security_acl.py

```diff
@@ -0,0 +1,176 @@
+        self.log.info("(3)==>Verify container permission %s", permission_type)
+        self.update_container_acl(
+            secTestBase.acl_entry(user_type, self.current_user, "rw"))
+        self.verify_cont_rw_attribute(
+            "write", "pass", attribute_name, attribute_value)
+        self.setup_container_acl_and_permission(
+            user_type, self.current_user, permission_type, cont_permission)
+        self.log.info(
+            "(3.1)Verify container_attribute: write, expect: %s", expect_write)
+        self.verify_cont_rw_attribute(
+            "write", expect_write, attribute_name, attribute_value)
+        self.log.info(
+            "(3.2)Verify container_attribute: read, expect: %s", expect_read)
+        self.verify_cont_rw_attribute("read", expect_read, attribute_name)
+
+        #(4)Verify container permissions tT rw-property
+        permission_type = "property"
+        self.log.info("(4)==>Verify container permission tT, rw-property")
+        self.log.info(
+            "(4.1)Update container-acl %s, %s, permission_type: %s with %s",
+            user_type, self.current_user, permission_type, cont_permission)
+        self.setup_container_acl_and_permission(
+            user_type, self.current_user, permission_type, cont_permission)
+        self.log.info(
+            "(4.2)Verify container_attribute: read, expect: %s", expect_read)
+        self.verify_cont_rw_property("read", expect_read)
+        self.log.info(
+            "(4.3)Verify container_attribute: write, expect: %s", expect_write)
+        self.verify_cont_rw_property(
+            "write", expect_write, property_name, property_value)
+        self.log.info(
+            "(4.4)Verify container_attribute: read, expect: %s", expect_read)
+        self.verify_cont_rw_property("read", expect_read)
+
+        #(5)Verify container permissions aA, rw-acl
+        permission_type = "acl"
+        self.log.info("(5)Verify container permission aA, rw-acl ")
+        self.log.info(
+            "(5.1)Update container-acl %s, %s, permission_type: %s with %s",
+            user_type, self.current_user, permission_type, cont_permission)
+        expect = "pass"  #User who created the container has full acl access.
+        permission_acl = ""
+        self.setup_container_acl_and_permission(
+            user_type, self.current_user, permission_type, cont_permission)
+        self.log.info("(5.2)Verify container_acl: write, expect: %s", expect)
+        self.verify_cont_rw_acl(
+            "write", expect, secTestBase.acl_entry(
+                user_type, self.current_user, cont_permission))
+        self.log.info("(5.3)Verify container_acl: read, expect: %s", expect)
+        self.verify_cont_rw_acl("read", expect)
+
+        #(6)Verify container permission o, set-owner
+        self.log.info("(6)==>Verify container permission o, set-owner")
+        permission_type = "ownership"
+        expect = "deny"
+        if "w" in cont_permission:
```

Comment:
should be "o"?

Reply:
"w" is read from test.yaml file (var cont_permission) which mux with 4 options: none, "r", "w", and "rw" for each of the rw, tT, aA and o (w only) tests.
On set-owner "o" test, test script perform the correct acl-set with "o".